### DISCUSSION TITLE
Added features for improved research quality

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#216] [Announcements]
We're excited to announce the latest release of GPT Researcher! This release includes an addition of various and powerful search engines for improved overall research quality and experience. 
1. Tavily API - An LLM focused search engine for optimized, explicit and factual results. It's free of use (!)
2. Serp API - Requires paid account
3. Google search API - Requires paid account
4. Searx - Meta search engine.

In addition, this release includes additional performance improvements and bug fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v0.0.7'>Added features for improved research quality</a>.</em>

**Comment by TomLucidor** [comment]
Pleas econtinue adding support for more API-free search engines e.g. SearXNG, DDG, StartPage, Bing, etc.
